tests/Oryx.Integration.Tests/LocalDockerTests/Fixtures/SqlServerDbContainerFixture.cs

```diff
@@ -53,7 +53,7 @@ protected override bool WaitUntilDbServerIsUp()
             {
                 try
                 {
-                    var lookUpText = "SQL Server is now ready for client connections";
+                    var lookUpText = "The default language (LCID 0) has been set for engine and full-text services.";
```

Comment:
This is the final message that is printed when the SQL Server is up and ready. Even though the earlier message says 'ready for client connections' I have received errors some times when setting up data.

Also regarding setting up data, one thing we could do is to bake the data into the image itself. This improves the perf a bit, but still does not avoid the problem of making requests to the container before it is ready.

Reply:
FYI:
![image](https://user-images.githubusercontent.com/6559784/57242800-55429200-6fe9-11e9-895a-919b176c84dc.png)